test/Microsoft.TemplateEngine.TestHelper/PackageManager.cs

```diff
@@ -110,7 +110,9 @@ public string PackNuGetPackage(string projectPath)
                 p.WaitForExit();
                 if (p.ExitCode != 0)
                 {
-                    throw new Exception($"Failed to pack the project {projectPath}");
+                    throw new Exception($"Failed to pack the project {projectPath}: " +
+                        $"{Environment.NewLine}Stdout: {p.StandardOutput.ReadToEnd()}." +
```

Comment:
This is sensitive code. If process p has not been configured for redirection it will [throw](https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.process.standarderror?view=net-5.0#exceptions).
I recommend to get stdout/err above in a try {} block.